examples/databaseSession.ts

```diff
@@ -0,0 +1,35 @@
+import { DatabaseSession, DatabaseType, TelegramClient } from "https://deno.land/x/grm/mod.ts";
+
+// Login and create an application on https://my.telegram.org
+// to get values for API ID and API Hash.
+const apiId = 123456;
+const apiHash = "abcd1234";
+
+// Fill in this later with the value from client.session.save(),
+// so you don't have to login each time you run the file.
+const dbSession = new DatabaseSession("sessionName", {
+  type: DatabaseType.SQLite,
+  adapterOptions: {
+    file: "examples/session.db",
+  },
+});
+
+console.log("Loading interactive example...");
+const client = new TelegramClient(dbSession, apiId, apiHash, {
```

Comment:
For simplicity in examples, consider removing the `clientParams` (4th argument) here.